modules/redis/redis-worker.h

```diff
@@ -40,6 +40,9 @@ typedef struct _RedisDestWorker
   gchar **argv;
   size_t *argvlen;
 
+  /*Batch mode*/
+  size_t num_of_messages_waiting_for_reply;
```

Comment:
In my understanding the `batch_size` will be distributed among the workers and that is why each worker has to know how many messages it sent so it can request as many replies.

Reply:
The `batch_size` is the property of `LogThrreadedDestWriter` thus it is unique to the specific worker. Check when and how it is manipulated in the `lib/logthrdest/logthrdestdrv.c` file.